eng/pipelines/ci-template.yml

```diff
@@ -108,14 +108,15 @@ steps:
       condition: and(succeeded(), ${{ parameters.unitTest }})
 
     - script: inv regenerate
-      displayName: "Regenerate Code"
+      displayName: "Regenerate Code(autorest)"
       workingDirectory: $(Build.SourcesDirectory)/autorest.python/packages/${{parameters.folderName}}/
       condition: and(succeeded(), ${{ parameters.regenerate }}, eq('${{parameters.folderName}}', 'autorest.python'))
 
     - script: |
         find test/azure/generated -type f ! -name '*apiview_mapping_python.json*' -delete
+        find test/unbranded/generated -type f ! -name '*apiview_mapping_python.json*' -delete
```

Comment:
the one thing i'm not sure about is serialization. If we can't edo serialization here, we should instead add similar functionality but for serialization (and we can rename `type` to `deserialization`) instead of adding extra logic through the msrest way

Reply:
- there is condition to do it only for typespec-python: https://github.com/Azure/autorest.python/blob/8e20f2136af3ae2e1939533a36c980b7c9bbe572/eng/pipelines/ci-template.yml#L121C12-L121C117
- If we add it in regenerate.ts, when regenreate fails, the local repo will be in a mess since all the genreated folders are already deleted.